CefSharp.Core/Internals/CefTaskScheduler.h

```diff
@@ -42,7 +43,11 @@ namespace CefSharp
                 }
                 catch (Exception^)
                 {
-
+                    // This should never ever happen.
```

Comment:
Probably something like that, but on both `Cef` and `CefAppWrapper` or something so we handle browser and render process cases. Probably via an interface/abstract C++ class.